views/pages/search.html

```diff
@@ -39,6 +39,19 @@
       {{#input-text}}surname{{/input-text}}
       {{#input-text}}forenames{{/input-text}}
       {{#input-text}}dob{{/input-text}}
+      <details{{#errors.dob}} open{{/errors.dob}}>
+        <summary><span class="summary">You can enter the date in any of these formats:</span></summary>
+        <div id="dob-extended-hint" class="panel panel-border-narrow">
+          <ul class="format-list">
+            <li>Full date format (with slashes): <span>31/12/2010</span></li>
+            <li>Full date format without leading zeros: <span>1/1/2010</span></li>
+            <li>Full date format with 2 digit year: <span>1/1/11</span></li>
+            <li>Short date format (without slashes): <span>31122010</span></li>
+            <li>Short date format must have leading zeros: <span>01012010</span></li>
+            <li>Short date format with 2 digit year: <span>010111</span></li>
```

Comment:
Is it worth picking a single date that we can use for all of them? 31/01/2010?

Reply:
Yes, Jeremy asked the same thing, but as discussed; there are some mutually exclusive date examples that wouldn't work with a single date example.